code/modules/goonchat/browserOutput.dm

```diff
@@ -189,14 +189,17 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("data/iconCache.sav")) //Cache of ic
 	var/list/partial = splittext(iconData, "{")
 	return replacetext(copytext(partial[2], 3, -5), "\n", "")
 
+GLOBAL_LIST_EMPTY(bicon_cache)
+
 /proc/bicon(thing)
 	if (!thing)
 		return
-	var/static/list/bicon_cache = list()
+	var/list/bicon_cache = GLOB.bicon_cache // now this isn't empty every time...

```

Comment:
set bicon_cache to the global bicon_cache

Reply:
why are you saving it to a var

Reply:
fixed